rules/windows/process_creation/proc_creation_win_susp_regsvr32_remote_share.yml

```diff
@@ -0,0 +1,25 @@
+title: Suspicious Regsvr32 Execution From Remote Share
+id: 88a87a10-384b-4ad7-8871-2f9bf9259ce5
+status: experimental
+description: Detects REGSVR32.exe to execute DLL hosted on remote shares
+references:
+    - https://thedfirreport.com/2022/10/31/follina-exploit-leads-to-domain-compromise/
+author: Nasreddine Bencherchali
+date: 2022/10/31
+tags:
+    - attack.defense_evasion
+    - attack.t1218.010
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\regsvr32.exe'
+        - OriginalFileName: '\REGSVR32.EXE'
+    selection_cli:
+        CommandLine|contains: ' \\\\'
+    condition: all of selection_*
+falsepositives:
+    - Unknown
+# Decrease to medium if this is something common in your org
```

Comment:
Or disable the rule altogether. I would say this is true for any rule, thus not a necessary comment.

Reply:
True. My mindset here was for some rules the line between high and medium is thin depending on the org. I just added it for people who perhaps use this (even though I don't think it's common)

Reply:
I don't mind it being there. It was just a general remark.